dali/pipeline/data/view_test.cc

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include "dali/pipeline/data/views.h"
+
+#define EXPECT_ENFORCE_FAIL(statement) EXPECT_THROW(statement, std::runtime_error)
```

Comment:
I suppose it should be undef'ed at the end of the file

Reply:
It is cc file. So unless we are planning to make one, big file that include others just to speed up the compilation it should not pollute anything.